p2p/sharding_peer.py

```diff
@@ -92,6 +89,7 @@ def handle_sub_proto_msg(self, cmd: Command, msg: protocol._DecodedMsgType) -> N
     async def get_collations(self,
                              collation_hashes: List[Hash32],
                              cancel_token: CancelToken) -> Set[Collation]:
+        from trinity.protocol.les.utils import gen_request_id
```

Comment:
cross-package utils import?

Reply:
temporary, will be removed once everything has been migrated.